demosplan/DemosPlanCoreBundle/Entity/Procedure/ProcedureSettings.php

```diff
@@ -202,28 +202,28 @@ class ProcedureSettings extends CoreEntity implements UuidEntityInterface, Proce
      *
      * @ORM\Column(name="_ps_designated_phase", type="string", length=50, nullable=true)
      */
-    protected $designatedPhase = null;
+    protected $designatedPhase;
 
     /**
      * @var string
      *
      * @ORM\Column(name="_ps_designated_public_phase", type="string", length=50, nullable=true)
      */
-    protected $designatedPublicPhase = null;
+    protected $designatedPublicPhase;
 
     /**
      * @var DateTime
      *
      * @ORM\Column(name="_ps_designated_switch_date", type="datetime", nullable=true)
      */
-    protected $designatedSwitchDate = null;
```

Comment:
see above

Reply:
I dont like what the style fixer does!
It wasn't me 4d62cca1cbf0553c05be7f209bee90ba2e429965